managed/services/telemetry/telemetry.go

```diff
@@ -135,15 +135,15 @@ func (s *Service) Run(ctx context.Context) {
 
 		report := s.prepareReport(ctx)
 
-		s.l.Debugf("\nTelemetry captured:\n%s\n", s.Format(report))
-
 		if s.config.Reporting.Send {
 			s.sendCh <- report
 		} else {
 			s.l.Info("Telemetry sent is disabled.")
 		}
 	}
 
+	s.l.Infof("\nTelemetry captured:\n%s\n", s.Format(s.prepareReport(ctx)))
```

Comment:
Do we want to print this to use logs with info level?

Reply:
Man, I mistakenly remove the wrong log command. Thanks, it's rectified.